websites/R/RadiosNet/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "784546644766621746",
+		"name": "ntn_ss"
+	},
+	"service": "RadiosNet",
+	"description": {
+		"en": "The most famous radio aggregator on the Brazilian Internet, active since 1997.",
+		"pt_BR": "O agregador de rádios mais famoso da internet brasileira, em atividade desde 1997."
+	},
+	"url": "radios.com.br",
+	"matches": [
+		"*://radios.com.br/*"
+	],
```

Comment:
Based on the code and visiting the website, the following urls should also be added:
* `www.radios.com.br`
* `play.radios.com.br`